plugins/source/digitalocean/resources/services/registries/repositories.go

```diff
@@ -0,0 +1,39 @@
+// Code generated by codegen; DO NOT EDIT.
+
+package registries
+
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func Repositories() *schema.Table {
+	return &schema.Table{
+		Name:     "digitalocean_registries_repositories",
```

Comment:
Similar comment applies to this one - ideally IMO the table name should be `digitalocean_registry_repositories`. That's at least the convention we've gone with in the AWS plugin